src/libPMacc/include/algorithms/math/doubleMath/trigo.tpp

```diff
@@ -57,6 +57,17 @@ struct Cos<double>
 };
 
 template<>
+struct ACos<double>
+{
+    typedef double result;
+
+    HDINLINE double operator( )(const double& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE double operator( )( const....)`